meta-balena-common/recipes-support/resin-device-progress/resin-device-progress/resin-device-progress

```diff
@@ -62,7 +62,7 @@ if [ -z "$REGISTERED_AT" ]; then
     exit 1
 fi
 
-if [ -z "${DEVICE_UUID}" ]; then
+if [ -z "${UUID}" ]; then
```

Comment:
You still seem to reference `${DEVICE_UUID}` at L73

Reply:
fixed